native_client/multistrap_raspbian64_bullseye.conf

```diff
@@ -0,0 +1,14 @@
+[General]
+arch=arm64
+noauth=false
+unpack=true
+bootstrap=Debian
+aptsources=Debian
+cleanup=true
+
+[Debian]
+packages=apt libc6 libc6-dev libstdc++-9-dev linux-libc-dev libffi-dev libpython3.9-dev libsox-dev python3-numpy python3-setuptools
+source=http://deb.debian.org/debian
```

Comment:
this file should be named `armbian64` since it is debian and not raspbian

Reply:
To be perfectly clear the name should be `multistrap_armbian64_bullseye.conf`

Reply:
@lissyx / @wasertech We already have a [conf file](https://github.com/coqui-ai/STT/blob/main/native_client/multistrap_armbian64_buster.conf) named that way, any suggestion?

Reply:
We should only have 4.

Two for armv7 buster and bullseye.

and two for aarch64 buster and bullseye.

We already had the buster ones for 3.7 (buster) so if we multiple configs for the same target, make sure they are the same and if so remove the duplicates.

Reply:
Agreed, it sounds like there's something wrong if you end up with duplicates.

Reply:
Fixed